ssz/src/main/java/org/ethereum/beacon/ssz/annotation/SSZ.java

```diff
@@ -74,12 +75,20 @@
   String type() default "";
```

Comment:
Yes, it still relevant, since you may have java `BigInteger` field but want it represent `uin256` SSZ type
Though some types can be removed for now, since SSZ now defines only `uintN` and `bool`